cocos/3d/CCBundle3DData.h

```diff
@@ -324,8 +324,8 @@ struct NTextureData
      std::string id;
      std::string filename;
      Usage type;
-     GLenum wrapS;
-     GLenum wrapT;
```

Comment:
Should be care about it, because the raw value type is not `backend::SamplerAddressMode`. Any code convert it from GLenum to `backend::SamplerAddressMode`?

Reply:
the original address mode value was stored as a string in files, the `parseSamplerAddressMode` will convert string value to `backend::SamplerAddressMode`. So it is fine to do so.